vip-plugins.php

```diff
@@ -10,3 +10,23 @@
  */
 
 require_once( __DIR__ . '/vip-plugins/vip-plugins.php' );
+
+/** loading helper and behaviors for Gutenberg on VIP Go **/
+
+/** load Gutenberg Ramp **/
+if ( file_exists( __DIR__ . '/ramp-for-gutenberg/ramp-for-gutenberg.php' ) ) {
+	require_once( __DIR__ . '/ramp-for-gutenberg/ramp-for-gutenberg.php' );
+}
+
+/** Turn off the UI for Ramp **/
+add_action( 'plugins_loaded', function() {
+	remove_action( 'admin_init', 'ramp_for_gutenberg_initialize_admin_ui' );
+} );
+
+/** Loading helper **/
+function wpcom_vip_load_gutenberg( $criteria ) {
+	if ( !function_exists( 'ramp_for_gutenberg_load_gutenberg' ) ) {
```

Comment:
Isn't this included as part of Ramp, which we load above?

Reply:
It is indeed ... I was thinking of a strange case where this function would be moved into another file, making it possible to call it without loading ramp, perhaps in a local environment or something?

Reply:
That said, there's no immediate need for it so I'd also be happy to remove it.